raft/log.go

```diff
@@ -1014,7 +1042,7 @@ func (l *Log) applier(done chan chan struct{}) {
 				l.tracef("applier: no entries")
 				return nil
 			} else if l.appliedIndex == l.commitIndex {
-				l.tracef("applier: up to date")
+				//l.tracef("applier: up to date")
```

Comment:
It was making a lot of noise. I think the `applier` trace above was good enough.